ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,118 @@
+            {
+                dictionary = AddCountryToDictionary(dictionary);
+                dictionary = EditIsTelenorSupportedInDictionary(dictionary);
+                PrintDictionary(dictionary);
+                fileReader.PrintFile(ConvertDictionaryToList(dictionary), filePath);
+            }
+            Console.WriteLine("End...");
+            Console.ReadLine();
+        }
+
+        public static Dictionary<int, Country> CreateDictionary(string filePath)
+        {
+           if (fileReader.GetCountryListFromFile(filePath) != null)
+            {
+                int i = 0;
+                List<string> countryList = fileReader.GetCountryListFromFile(filePath);
+                Dictionary<int, Country> countries = new Dictionary<int, Country>();
+                string subline = ";";
+                int startIndex = 0;
+                foreach (string countryElement in countryList)
+                {
+                    int index = countryElement.IndexOf(subline);
+                    Country country = new Country
+                    {
+                        Name = countryElement.Substring(startIndex, index)
+                    };
+                    if (countryElement.Substring(index + 1).ToLower() == "true")
+                    {
+                        country.IsTelenorSupported = true;
+                    }
+                    else
+                    {
+                        country.IsTelenorSupported = false;
+                    }
+                    countries.Add(i + 1, country);
+                    i++;
+                }
+                return countries;
+            }
+            return null;
+        }
+
+        public static Dictionary<int, Country> AddCountryToDictionary(Dictionary<int, Country> dictionary)
+        {
+            bool flag = false;
+            foreach (KeyValuePair<int, Country> kvp in dictionary)
+            {
+                if ((kvp.Value.Name == "Ukraine"))
+                {
+                    flag = true;
+                }
+            }
+            if (flag == true)
+            {
+                return dictionary;
+            }
+            else
+            {
+                Country country = new Country() { 
+                    Name = "Ukraine", 
+                    IsTelenorSupported = true };
+
+                dictionary.Add(dictionary.Count + 1, country);
+                return dictionary;
+            }
+        }
+
+        public static void PrintDictionary(Dictionary<int, Country> dictionary)
+        {
+            Console.WriteLine("\nHere is the Dictionary: ");
+            foreach (KeyValuePair<int, Country> kvp in dictionary)
+            {
+                Console.WriteLine("Key = {0}, Value Name = {1}, Value IsTelenorSupported = {2}", kvp.Key, kvp.Value.Name, kvp.Value.IsTelenorSupported);
+            }
+        }
+
+        public static Dictionary<int, Country> EditIsTelenorSupportedInDictionary(Dictionary<int, Country> dictionary)
```

Comment:
same as with Ukraine. could you please create a method for any country and call it twice for each country you need?